src/index.ts

```diff
@@ -21,15 +89,15 @@ import FileSystem from "./fs/index.js"
  * See `loadFileSystem` if you want to load the user's file system yourself.
  * NOTE: Only works on the main/ui thread, as it uses `window.location`.
  *
- * The `useWnfs` option is only necessary for apps that create their own WNFS. Apps that
- * request filesystem `permissions` from the auth lobby can ignore `useWnfs` in this version
- * of webnative.
  */
-export async function initialise(options: InitOptions): Promise<State> {
+export async function fissionApp(options: LinkedAppInitOptions): Promise<LinkedAppState> {
```

Comment:
Renaming of the existing initialize function. 

Not sure about the name! How do we name an app to make it clear that it uses the Fission Auth Lobby?

Reply:
What makes an app a Fission app? An AOL app still registers a user with the Fission servers and stores data on it, so I guess that's a Fission app too?

Maybe we name this according to what the lobby gives us? The lobby serves as a "safety zone", in that people can trust the lobby so that it can provide (limited) access to subsets of their data to other apps. You could replicate that behaviour in different ways, but this function expects a specific way of doing so.

Maybe `lobbyApp` makes sense? A lobby app is an app that redirects to a root authority app and comes back with all necessary UCANs and keys to provide our app access to the data, this in the specific lobby format.

That makes me think, what if you have two AOL apps, where one delegates to the other. One would the root authority, and the second wants to have access to a subset of the user's data. How would that process work? Does (1) "upgrade" to the Fission lobby, or does it serve as something similar to the lobby for (2)? What would the init function be called for app (2)?

Sorry, thinking out loud 😅

Reply:
Yeah, that's a great point! AOL apps are still Fission apps when they use the Fission server.

`lobbyApp` does seem like a better name. This function could still set some other backend by setting endpoints, so this should be more generic. Though it might be confusing because should a `lobbyApp` be a lobby?

Another possibility is `linkedApp`, which hints at the fact that it needs needs to be linked by some app with root authority. (Though in theory, it could be linked by any app with equal or greater authority. Future possibilities here of course 😅 .)

> That makes me think, what if you have two AOL apps, where one delegates to the other. One would the root authority, and the second wants to have access to a subset of the user's data. How would that process work? Does (1) "upgrade" to the Fission lobby, or does it serve as something similar to the lobby for (2)? What would the init function be called for app (2)?

(1) would use the `app` initialization function which would start it off with root authority, and (2) would use `lobbyApp`/`linkedApp`/`someOthername` to initialize with permissions to request from (1). And yeah, (1) has the same level of authority as the auth lobby, so it could be used to serve that role (after we've AOL app linking).

Reply:
Another possibility, `permissionedApp`? Or maybe there is something else along those lines.

Reply:
> (1) would use the ...

Yes exactly 👍 I do like the sound of `permissionedApp`, with it having an option called `permissionedBy: URL` option 🤔

Reply:
Yeah for sure! Not on this PR, but we could refactor the way that endpoints are handled so they passed into initialization functions instead of set globally. That would make it explicit who a `permissionedApp` is `permissionedBy`.

Let's change this to `permissionedBy`, and if we come up with something else we can change again.